source/Calamari/Kubernetes/ResourceStatus/ResourceRetriever.cs

```diff
@@ -72,27 +74,36 @@ private IEnumerable<Resource> GetChildrenResources(Resource parentResource, IKub
                             }).ToList();
         }
 
-        T TryParse<T>(Func<string, Options, T> function, string jsonString, Options options)
+        T TryParse<T>(Func<string, Options, T> function, ICommandOutput commandOutput, Options options)
         {
+            var jsonString = commandOutput.InfoLogs.Join(string.Empty);
             try
             {
+                
                 return function(jsonString, options);
             }
             catch (JsonException)
             {
-                LogJsonStringError(jsonString, options);
+                LogJsonStringError(jsonString, commandOutput, options);
                 throw;
             }
         }
 
-        void LogJsonStringError(string jsonString, Options options)
+        void LogJsonStringError(string jsonString, ICommandOutput commandOutput, Options options)
         {
             if (options.PrintVerboseKubectlOutputOnError)
             {
                 log.Error("Failed to parse JSON:");
                 log.Error("---------------------------");
                 log.Error(jsonString);
                 log.Error("---------------------------");
+                log.Error("Full command output:");
+                log.Error("---------------------------");
+                foreach (var msg in commandOutput.Messages)
+                {
+                    log.Error($"{msg.Level}: {msg.Text}");
+                }
+                log.Error("---------------------------");
```

Comment:
Write the entire command output so we can see if anything is erroneously ending up in the error stream